docs/examples/robust_facility.ipynb

```diff
@@ -32,12 +32,20 @@
     "With this information in mind, our optimization problem becomes:\n",
     "$$\n",
     "\\begin{array}{ll}\n",
+<<<<<<< HEAD
```

Comment:
You need to fix this (also in L42, L48. In short, you need to decide what stream you would like and keep that stream).